Index: CMakeLists.txt
===================================================================
--- CMakeLists.txt
+++ CMakeLists.txt	2018-05-22 03:40:05.056082972 +0200
@@ -72,7 +72,7 @@
 	message(WARNING "You are cross-compiling. Skipping IEEE 754 test.")
 endif()
 
-find_package(SDL2 2.0.4 REQUIRED)
+find_package(SDL2 2.0 REQUIRED)
 find_package(Crypto 1.0 REQUIRED)
 find_package(Boost 1.50 REQUIRED COMPONENTS iostreams program_options regex system thread random)
 
